#! /bin/sh /usr/share/dpatch/dpatch-run
# 28_bconsole-v3-missing-edited-consistency-opts.manpage.dpatch
#
# Description: adds missing and new options in the bconsole utility's
#  manpage. Also introduces better consistency between the Bacula
#  utilities manpages.
# Origin: vendor, file:///dev/null
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=499619
# Forwarded: yes
# Author: Lucas B. Cohen <lbc@members.fsf.org>
# Last-Update: 2009-12-07
#
# created by Lucas B. Cohen <lbc@members.fsf.org> with dpatch v2.0.31
@DPATCH@
diff --git a/bacula/manpages/bconsole.8 b/bacula/manpages/bconsole.8
index 843712d..c940545 100644
--- a/bacula/manpages/bconsole.8
+++ b/bacula/manpages/bconsole.8
@@ -2,7 +2,7 @@
 .\" First parameter, NAME, should be all caps
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
-.TH CONSOLE 8 "26 September 2009" "Kern Sibbald" "Network backup, recovery and verification"
+.TH CONSOLE 8 "4 December 2009" "Kern Sibbald" "Network backup, recovery and verification"
 .\" Please adjust this date whenever revising the manpage.
 .\"
 .SH NAME
@@ -25,13 +25,16 @@ Specify configuration file.
 Set debug level to \fInn\fP.
 .TP
 .BI \-dt
-Print a timestamp in debug output.
+Print timestamp in debug output.
+.TP
+.B \-n
+No conio (for debugging).
 .TP
 .B \-s
-No signals.
+No signals (for debugging).
 .TP
 .B \-t
-Test config mode: read configuration and exit.
+Test the configuration file and report errors.
 .TP
 .B \-?
 Show version and usage of program.